lib/phoenix_live_view/tag_engine.ex

```diff
@@ -812,7 +812,18 @@ defmodule Phoenix.LiveView.TagEngine do
   defp handle_tag_attrs(state, meta, attrs) do
     Enum.reduce(attrs, state, fn
       {:root, {:expr, _, _} = expr, _attr_meta}, state ->
-        handle_attrs_escape(state, meta, parse_expr!(expr, state.file))
+        ast = parse_expr!(expr, state.file)
+
+        if pairs = dynamic_attrs_literal(ast) do
+          # Optimization: if keys are known at compilation time, we
+          # inline the dynamic attributes
+          Enum.reduce(pairs, state, fn {key, value}, state ->
+            name = to_string(key)
```

Comment:
We should probably add some tests to check for dash. I think atoms are dashrized, strings are not, but please double check. We should mirror the behaviour here. :D

Reply:
We no longer dasherize atoms, that's the change we made some time ago :D

https://github.com/phoenixframework/phoenix_live_view/blob/00713d4c287263a20e8a413124e4b6d6ca337597/lib/phoenix_live_view/tag_engine.ex#L925-L927